types/errors/Error.d.ts

```diff
@@ -0,0 +1,27 @@
+import { TErrorCode } from './ErrorCode';
```

Comment:
# Question

Why is this directory lowercased? Is this a requirement for Typescript or simply missed when added?

Reply:
Not a requirement for TS, but pretty common to make directories lowercased.

Reply:
@amatthews4851 have any thoughts on this by chance?